conf/plants/allowed_species.json

```diff
@@ -72,6 +76,9 @@
     "prunus_dulcis",
     "prunus_persica",
     "rosa_chinensis",
+    "saccharomyces_cerevisiae",
+    "saccharum_spontaneum",
```

Comment:
Following conversation in Slack with this addition - this puts us at something like 103 genomes - over the 100 allowed in frozen release - do we care? Or can we allow a little flexibility for this?

Reply:
Yes, we do. Will remove it from here and `mlss_conf.xml` now.